docs/docs/backup-restore.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Backup & Restore
+---
+
+# Backup & Restore
+
+Marathon 1.5 and later has backup and restore functionality built in.
+The complete state from the persistent data store can be backed up to an external file on the local disk or to an external storage provider.
+A backup can be restored again and will bring back the exact same state as it was available at the time of backup creation.  
+
+## Limitations
+
+Marathon is only able to backup the internal state of Marathon itself. 
+A backup will **not** capture the state of any other external system.
+This is specifically true for Mesos, which has its own backup & restore functionality.
+To create a backup of your cluster, a backup of all the system components is needed.
+
+Restoring a backup in Marathon without restoring state in other system components can lead to unwanted behavior.
+Tasks that have been started by Marathon after the backup has been created, will be unknown to Marathon after restoring and will be killed.
+Tasks that have been killed by Marathon after the backup has been created, will be expunged during the next reconciliation.
+All changes to apps, groups, and pods after the backup has been created, are lost if this backup is restored.
+
+## Backup
+
+A backup operation reads the existing state from Marathon in zk and writes this data to a storage location.
+This data contains all entities under `zk://host:port/path/state/`. 
+It will not backup the state of the leader election: `zk://host:port/path/leader/`
+
+A backup is a single file in [tar](http://www.gnu.org/software/tar/) format, which contains all elements as file entries.
+
+### Options to create a backup 
+
+There are multiple ways to create a backup:
+
+- cmd line flag `--backup_location {URI}`
+Marathon can be started with the command line option: `--backup_location` to a configurable URI. 
+This option is especially useful and strongly recommended to set, since it will create a backup 
+of the Marathon internal state, whenever Marathon performs a migration. 
+A migration usually happens during an upgrade of Marathon.
+
+- API endpoint `DELETE /v2/leader?backup={URI}` (experimental)
+It is possible to create a backup at any point in time using the leader endpoint.
+Using this endpoint will force the current running leader to abdicate.
```

Comment:
The API call above will force the current leader to abdicate. The next leader will create a backup before it takes over leadership.

This method is experimental: the API endpoint may change in the future.